pulsar-client-cpp/python/pulsar/__init__.py

```diff
@@ -403,6 +405,8 @@ def __init__(self, service_url,
           Configure whether the Pulsar client validates that the hostname of the
           endpoint, matches the common name on the TLS certificate presented by
           the endpoint.
+        * `logger`:
+          Set a Python logger for this Pulsar client.
```

Comment:
Perhaps spell out explicitly that this must be an instance of `logging.Logger`?

Reply:
Fixed